string groupName
string droneName
---
string result
